store/sqlstore/team_store.go

```diff
@@ -678,14 +678,31 @@ func (s SqlTeamStore) GetMember(teamId string, userId string) (*model.TeamMember
 	return dbMember.ToModel(), nil
 }
 
-func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, restrictions *model.ViewUsersRestrictions) ([]*model.TeamMember, *model.AppError) {
+func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, restrictions *model.ViewUsersRestrictions, teamMembersGetOptions *model.TeamMembersGetOptions) ([]*model.TeamMember, *model.AppError) {
 	query := s.getTeamMembersWithSchemeSelectQuery().
 		Where(sq.Eq{"TeamMembers.TeamId": teamId}).
 		Where(sq.Eq{"TeamMembers.DeleteAt": 0}).
-		OrderBy("UserId").
 		Limit(uint64(limit)).
 		Offset(uint64(offset))
 
+	if teamMembersGetOptions != nil {
+		if teamMembersGetOptions.Sort == "Username" || teamMembersGetOptions.ExcludeDeletedUsers {
```

Comment:
Should we pull out `"Username"` into a `model` constant?

Reply:
Do you have a suggestion for what to call the constant?
`USERNAME` is okay? and then we can call it by `model.USERNAME` ? 
Not certain if this is appropriate or if you had other thoughts.

Reply:
To add to above, does it make sense to make `"UserId"` into a model constant or no? 
Also, should the `USERNAME` constant be in the user model file or team member model file?